source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -224,12 +229,38 @@ AdaptiveNamespaceStart
     [contract(InternalContract, 1)]
     [internal]
 #endif
-    typedef [v1_enum] enum HAlignment
+    typedef [v1_enum] enum _HorizontalAlignment
```

Comment:
>_ [](start = 27, length = 1)

Lets not include this underscore here or in VerticalAlignment

Reply:
So, I don't want to but it get's mad when the underscore isn't there. I get `Error MIDL2003 : redefinition : HorizontalAlignment_Left` and the same for all of the other values. Same with VerticalAlignment. This is because of the Windows::UI::Xaml::HorizontalAlignment.

---
In reply to: [253176897](https://github.com/Microsoft/AdaptiveCards/pull/2316#discussion_r253176897) [](ancestors = 253176897)

Reply:
Ugh, that's probably why it was called HAlignment in the first place. Was there a particular reason we changed the name? (other than HAligment being obviously not-ideal) Since this is what's going to be used by API callers i think we really don't want that underscore in there. 

---
In reply to: [254501325](https://github.com/Microsoft/AdaptiveCards/pull/2316#discussion_r254501325) [](ancestors = 254501325,253176897)

Reply:
Haha. Really just `HAlignment` being not-ideal. I also figured that we'd want HorizontalAlignment and VerticalAlignment to follow the same naming convention (so not `HAlignment` and `VerticalAlignment`). I can do `HAlignment` and `VAlignment` but I feel like having them be one letter off is kinda a problem. 

---
In reply to: [257419249](https://github.com/Microsoft/AdaptiveCards/pull/2316#discussion_r257419249) [](ancestors = 257419249,254501325,253176897)